packages/cli/e2e/vitest.setup.ts

```diff
@@ -0,0 +1,16 @@
+import { beforeAll, expect } from 'vitest';
+import { executeCLI } from './utils/command.js';
+
+const refreshToken = process.env.E2E_REFRESH_TOKEN;
+
+beforeAll(async () => {
```

Comment:
issue: doing this without an `afterAll` that logs out at the end might cause issues in local development, let's add that to be extra safe.